### PR TITLE
Openroad settings

### PR DESCRIPTION
**Pull request by oharboe**
Allows a non-source controlled settings.mk file locally that avoids having to write long make command lines with lots of arguments

### CODE REVIEW

**Comment by oharboe** [comment]
@maliberty Ready for review, the failures I saw were not related to this PR as such.

File: flow/Makefile [hunk 1]
```diff
@@ -1,88 +1,7 @@
-# ==============================================================================
-# Uncomment or add the design to run
-# ==============================================================================
+# settings.mk is not under source control. Put variables into this
+# file to avoid having to adding the to the make command line.
+-include settings.mk
 
-# DESIGN_CONFIG=./designs/nangate45/aes/config.mk
-# DESIGN_CONFIG=./designs/nangate45/ariane136/config.mk
-# DESIGN_CONFIG=./designs/nangate45/black_parrot/config.mk
```
Comment (maliberty) [inline flow/Makefile]: We've already discussed not removing these previously.  Please restore them.

Comment (oharboe) [reply flow/Makefile]: Will do. Do you mind if I force push, so I can simply delete this commit? Otherwise there will be delete + revert commit.

Comment (maliberty) [reply flow/Makefile]: Ok as I haven't looked further.

**Comment by oharboe** [comment]
@maliberty Cleaned up and on top of #1089 to avoid merge conflicts later.

**Review by maliberty — APPROVED** [review]

**Pull request merged**